vagrant/provisioning/roles/ldap-portal/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+- name: LDAP required packages
+  become: yes
+  yum:
+    state: installed
+    name: 
+      - openldap-clients
+      - python-ldap
+
+- name: install pyOpenSSL
+  become: yes
+  pip:
+    name: pyOpenSSL
+
+- name: list certs in the key store
+  become: yes
+  command: keytool -v -list -keystore "{{ java_trust_store }}" -storepass "{{ java_trust_store_pass }}"
+  register: cert_list
+  changed_when: false
+
+- name: import the ldap cert to the ArkCase trust store
+  become: yes
+  java_cert:
+    cert_alias: "arkcase_ldap_cert"
+    cert_url: "{{ ldap_portal_host }}"
+    cert_port: "{{ ldap_portal_port|int }}"
+    keystore_path: "{{ java_trust_store }}"
+    keystore_pass: "{{ java_trust_store_pass }}"
+    state: present
+  when: "'arkcase_ldap_cert' not in cert_list.stdout"
```

Comment:
Use a different alias here, we can't use `arkcase_ldap_cert` as alias for both the platform LDAP and the portal LDAP.